README.md

```diff
@@ -51,3 +55,28 @@ jobs:
         env:
           GH_PAT: ${{ secrets.GH_PAT }}
 ```
+
+### Example with User Data
+
+```yaml
+- name: Create self-terminating runner
+  id: aws-start
+  uses: omsf/start-aws-gha-runner@v1.0.0
+  with:
+    aws_image_id: ami-0f7c4a792e3fb63c8
+    aws_instance_type: g4dn.xlarge
+    aws_home_dir: /home/ubuntu
+    aws_userdata: |
+      #!/bin/bash
+      # Configure instance to terminate on shutdown
+      INSTANCE_ID=$(ec2-metadata --instance-id | cut -d " " -f 2)
+      aws ec2 modify-instance-attribute \
```

Comment:
Ok, have answered most of my own questions, the only one I can't seem to track down is: what makes sure that the instance actually has perms to execute this `aws ec2 modify-instance-attribute` command?

Reply:
Good call, this example was out of date, this block is not necessary for callers anymore, the instance is launched with `"InstanceInitiatedShutdownBehavior": "terminate"`.